### PR TITLE
Fix resource search

### PR DESCRIPTION
**Pull request by Trial97**
fixes: https://discord.com/channels/1031648380885147709/1303585964253249556/1304478773780942932
and it was introduced here: https://github.com/PrismLauncher/PrismLauncher/commit/803e26a4018d788178032b653fd06d1b8970b3a4#diff-99a0c396e944fade5391d69854c4b0c9cd9eecfe00b636c84c1159cab520a45bL92

### CODE REVIEW

**Review by Ryex — APPROVED** [review]

**Review by TheKodeToad — APPROVED** [review]
IDK how I ended up removing that

File: CONTRIBUTING.md
Comment (TheKodeToad) [inline CONTRIBUTING.md:1]: Wait NVM please drop this commit

**Review by TheKodeToad — DISMISSED** [review]

File: CONTRIBUTING.md
Comment (Trial97) [reply CONTRIBUTING.md:1]: Why? Should I do that?

Comment (Trial97) [reply CONTRIBUTING.md:1]: Like I know it bleed from other PR but still I wanted to test the linting of markdown

**Review by TheKodeToad — APPROVED** [review]

**Pull request merged**